jupyter-docker/Dockerfile

```diff
@@ -127,6 +127,38 @@ RUN apt-get install -yq --no-install-recommends \
  && pip install matplotlib-venn \
  && pip install "https://github.com/all-of-us/workbench/archive/pyclient-v0-1-rc3.zip#egg=aou_workbench_client&subdirectory=client/py"
 
+
+# Install Python 3 Kernel
+RUN apt-get update \
```

Comment:
Minor, is `apt-get update` needed given that we already did it above?

Reply:
Oh, it looks like the previous layer cleans up apt caches:
```
&& apt-get clean \
&& rm -rf /var/lib/apt/lists/* \
```

Maybe remove that above, and only do that after all the `apt-get`s are done.

Reply:
The purpose of keeping it in the same line is to keep the layer small.  If the cleaning is done in a different step, the extra apt stuff remains in the earlier layer.